open-company/open-company-web
# Fullscreen cleanup

Card: https://trello.com/c/k14gvnNo

Fullscreen post component has still all the editing related code. Also there are still problems when editing from a fullscreen post since cmail is rendered below the fullscreen post.

Also from: https://docs.google.com/document/d/1BNtz1uMJnqNablvaThGUzOFu7zEEcQoeNsoh3O1ry3M/edit
Item:
> When Iacopo shared a post with me, the fixed header was messed up near the emoji

To test:
- share a post via url with your team
- paste the url in the bar
- edit the post you see
- [x] do you see the cmail above the fullscreen post? Good
- now edit something
- click X out of Cmail
- [x] do you see the alert modal telling you that you are going to lose all your edits? Good
- cancel
- SAVE the changes
- [x] are you still in fullscreen? Good
